app/views/registrations/school_profile/show.html.erb

```diff
@@ -1,13 +1,19 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: registrations_question_two_path) %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">What is your school Unique Reference Number(URN)?</h2>
     <%= form_with(model: @school_profile_form, url: registrations_school_profile_path) do |f| %>
       <%= f.govuk_error_summary %>
-      <label class="govuk-hint">
-        This is a 6 digit number. Not sure? <a target='_blank' href='https://www.get-information-schools.service.gov.uk/'>Search for your school</a> on our register.
-        <%= f.govuk_text_field(:urn, width: 20, label:  nil, link_errors: true) %>
-      </label>
+      <%= f.govuk_text_field(
+            :urn,
+            width: 20,
```

Comment:
I'd rather use govuk rows and columns classes to get the width right - unless 20 (characters? percent?) is explicitly what we want.

Reply:
This adds the `govuk-input--width-20` to the outputted markup and seems to be the recommended way of doing it: https://govuk-form-builder.netlify.app/form-elements/text-field/#generating-text-inputs-with-custom-widths

Reply:
Right, and is urn 20 characters long?

My guess is that 20 is what looks like designs, rather than specifies expected length exactly - and if we want exact length, umm, maybe it should be like 6? And if we don't want the exact length, then maybe let's not pretend that we do.

Reply:
Right, got you!

There's no 6 character width, just 5 and 10. Shall I change it to 10?

Reply:
I guess. Check with Al or Shani?

Reply:
I think if we're doing it for design reasons it should just be fractional, right?

Reply:
Question is, did our designers think about the width here, or did they added a field that kind of look good - I would definitely double check with them.

Reply:
If it involves roping someone else in—especially as they sound like they're super busy—I'd prefer to leave it for another ticket as it isn't actually related to this PR so much—the width hasn't changed in this PR.

Are you okay with that?

Reply:
Sure